includes/traits/llms-trait-user-engagement-type.php

```diff
@@ -26,6 +26,32 @@ trait LLMS_Trait_User_Engagement_Type {
 	 */
 	protected $engagement_type;
 
+	/**
+	 * Returns the number of user engagements that have been awarded from the template.
+	 *
+	 * @since [version]
+	 *
+	 * @param int $template_id The post ID of the template.
+	 * @return int
+	 */
+	protected function count_awarded_engagements( $template_id ) {
+
+		$awarded_engagements_query = new LLMS_Awards_Query(
+			array(
+				'fields'    => 'ids',
+				'templates' => $template_id,
+				'per_page'  => 1,
+				'status'    => array(
+					'publish',
+					'future',
+				),
```

Comment:
```suggestion
					'future',
					'draft',
				),
```
I think we want this too now.
At the beginning we just didn't allow for `draft` awards.